sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringOperations.scala

```diff
@@ -119,6 +119,10 @@ trait CaseConversionExpression extends ExpectsInputTypes {
 /**
  * A function that converts the characters of a string to uppercase.
  */
+@ExpressionDescription(
+  usage = "_FUNC_(str) - Returns str with all characters changed to uppercase",
+  extended = """> SELECT _FUNC_('SparkSql') FROM src LIMIT 1;
```

Comment:
we support `SELECT _FUNC_('SparkSql')` directly. Not need to add "FROM src LIMIT 1"